### PR TITLE
Refactor crud

### PR DESCRIPTION
**Pull request by malevska**

### CODE REVIEW

File: src/components/PlayersComponent.tsx [hunk 1]
```diff
@@ -1,25 +1,22 @@
...
+import { PlayQuizComponent } from "./PlayQuizComponent";
 
 // Modifies the players list
 export const PlayersComponent = (props: {
   quiz: Quiz;
   index: string;
-  addPlayers: AppMethods["addPlayers"];
-  editPlayers: AppMethods["editPlayers"];
+  setPlayers: AppMethods["setPlayers"];
+  onClose: () => void;
```
Comment (skid) [inline src/components/PlayersComponent.tsx]: Maybe pass a single prop: `onFinish: ( players: string[] ) => void` and in the parent comp you do something like: `... onFinish={ ( players ) => setPlayers(players); setShowPlayers(false) }`.

Like this the players component doesn't have to care about closing itself.

File: src/components/PlayersComponent.tsx [hunk 2]
```diff
@@ -1,25 +1,22 @@
...
 }) => {
   const quiz = props.quiz;
   const [newPlayerName, setNewPlayerName] = useState("");
-  const [players, setPlayers] = useState<string[]>(props.quiz.players);
-  const history = useHistory();
+  const [playersList, setPlayersList] = useState<string[]>(props.quiz.players);
 
   const onClickHandler = () => {
```
Comment (skid) [inline src/components/PlayersComponent.tsx]: put a better name. onClickHandler for what? call it `addNewPlayer`

**Review by skid — APPROVED** [review]
Looks good to me. Comments are FYI mostly.

File: src/components/PlayersComponent.tsx (hunk 2 above)
Comment (malevska) [reply src/components/PlayersComponent.tsx]: or remove it completely, since is not necessary.

**Pull request merged**